SB/SB/Program.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using System.Text;
+using CsvHelper;
+using NLog;
+using NLog.Config;
+using NLog.Fluent;
+using NLog.Targets;
+
+namespace SB
+
+{
+    
+    internal class Program
+    {
+        private static readonly ILogger logger = LogManager.GetCurrentClassLogger();
+        
+        public static void Main(string[] args)
+        {
+            var config = new LoggingConfiguration();
+            var target = new FileTarget { FileName = @"C:\Work\Logs\SupportBank.log", Layout = @"${longdate} ${level} - ${logger}: ${message}" };
+            config.AddTarget("File Logger", target);
+            config.LoggingRules.Add(new LoggingRule("*", LogLevel.Debug, target));
+            LogManager.Configuration = config;
+            logger.Info("Starting program");   
+           
+            
+            var pathToCvs = "C:\\Work\\Bootcamp\\SB\\SB\\DodgyTransactions2015.csv";
+            bool running = true;
+            while (running)
+            {
+                string input;
+                input = Console.ReadLine();
+                string[] split = input.Split(' ');
+                if (split[0] == "exit")
+                {
+                    running = false;
+                }
+                else if (split.Length == 2 && split[0] == "List" && split[1] == "All")
+                {
+                    ListAll(pathToCvs);
+
+                }else if (split.Length == 3 && split[0] == "List")

```

Comment:
What happens if a person has 3 names? You can do `split.Length >= 2` and `split.Take(1).Join(' ')`